### DISCUSSION TITLE
Some settings are unclear to me

### DISCUSSION THREAD

**Discussion by oawd** [Rem0o/FanControl.Releases#2268] [Q&A]
Hello,
as a beginner in FanControl there are some settings in this software that are not clear to me and I did not find explanations in the documentation.

The first thing I do not understand is whether I should use the NvApiWrapper or not. In my opinion there should be a comparison between the NvApiWrapper and LHM, so that the new user undestands this setting to some extent.

In the fan control module it is not clear to me what "Offset" and "Minimum %"  are doing. "Offset" does add a percentage to every percental value and "Minimum %" is the lowest possible percental value for this fan, right?

Furthermore I was seeking for an option to set the delta value for the fan speed. That is the minimum change of the fan speed percental value, which has to be reached for the speed of the fan to actually get changed. This prevents the fan from changing its speed in 1% steps. It seems that this feature is not available in FanControl?

**Comment by Rem0o** [comment]
> The first thing I do not understand is whether I should use the NvApiWrapper or not. In my opinion there should be a comparison between the NvApiWrapper and LHM, so that the new user undestands this setting to some extent.

You are right there is no documentation on this.  I will put that on my to-do list.   Short version, NvAPIWrapper is much lighter on ressources, may miss hotspot/mem junction temp.

> In the fan control module it is not clear to me what "Offset" and "Minimum %" are doing. "Offset" does add a percentage to every percental value and "Minimum %" is the lowest possible percental value for this fan, right?

![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/b82c5b48-1fcd-42df-a352-ded4ef95893b)
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/1a8ae10a-45aa-48db-83e0-5b65973d7205)

> Furthermore I was seeking for an option to set the delta value for the fan speed. That is the minimum change of the fan speed percental value, which has to be reached for the speed of the fan to actually get changed. This prevents the fan from changing its speed in 1% steps. It seems that this feature is not available in FanControl?

This is not to be applied on the control itself, but on the fan curves you'll be creating.  The settings are hysteresis and response time, and have this effect.

Hope that cover your questions.

**Reply by oawd** [reply]
Yes, the settings are now clearer to me. Thanks.

You seem to have a different approach to this effect with hysteresis and response time. I will try it out,
